npc/re/quests/eden/eden_quests.txt

```diff
@@ -1855,38 +1855,37 @@ moc_para01,112,96,5	script	Administrator Michael	4_M_RUSMAN1,{
 				close;
 			}
 			mes "[Michael]";
-			if (Class == Job_Swordman || Class == Job_Swordman_High || Class == Job_Knight || Class == Job_Crusader) {
+			if (BaseClass == Job_Swordman) {
 				callsub S_Select,"We have one and Two-handed swords",
 					P_Slayer1,"Eden Slayer I","Two-handed sword","Attack 162","",
 					P_Sabre1,"Eden Saber I","One-handed sword","Attack 147","This is what we strive for.";
 			}
-			if (Class == Job_Thief || Class == Job_Thief_High || Class == Job_Assassin || Class == Job_Rogue || Class == Job_Ninja || Class == Job_Novice || Class == Job_Novice_High) {
+			else if (BaseClass == Job_Thief || BaseClass == Job_Ninja || BaseClass == Job_Novice) {
 				callsub S_Select,"",
 					P_Dagger1,"Eden Dagger I","Dagger","MATK+60, attack 124","This is what we strive for.";
 			}
-			if (Class == Job_Merchant || Class == Job_Merchant_High || Class == Job_Blacksmith || Class == Job_Alchemist) {
+			else if (BaseClass == Job_Merchant || Class == Job_Merchant_High || Class == Job_Blacksmith || Class == Job_Alchemist) {
```

Comment:
probably mislook or my PR messed up due to some issue during rebase this morning. Will try update it soon.